docs/react-v9/contributing/api-extractor.md

```diff
@@ -0,0 +1,25 @@
+### What is API Extractor?
+
+[API Extractor](https://api-extractor.com/) generates an api.md file containing the public API for a package. `@fluentui/react` and related packages as well as `@fluentui/web-components` use API Extractor to ensure that API changes must be reviewed before being merged into master. (`@fluentui/react-northstar` and related packages do not use API Extractor as of writing.)
+
+### Build Tasks
+
+As part of a package's build process, the api-extractor task checks the package's public API with the package's auto-generated .api.md file.
+
+If building locally, the API file will be updated automatically (in `7.0` and `master`). Be sure to commit the updates with your other changes!
+
+In PR builds, the task will fail if it detects missing API file updates.
+
+(Previously it was necessary to run a separate command `yarn update-api` to update the API files, but we decided to streamline this and do the update automatically as part of the build instead.)
+
+To generate the API markdown file, first commit your changes, then run:
+
+`yarn nx run <package-name>:generate-api`
+
+On Windows systems, you may have to run:
+
+`yarn build`
```

Comment:
I don't understand why is this necessary ? you dont need to run build on whole monorepo no matter the OS.

Reply:
The specific generate API command you've mentioned fails half the time on my machine/OS. This is the only other way I've been able to generate an API change file.

Reply:
Yep, failed to me as well when I tried to run it inside `react-image` component in v9, but it was enough to run only on the package I got error, In my case it was `@fluentui/react-shared-contexts`.

Reply:
@Hotell nvm, should be ok now, I might run it without `nx` that time 🥲

Reply:
Removed this whole bit :-)